graylog2-web-interface/src/components/cluster-configuration/ClusterConfigurationListView.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+
+import { Table, Label } from 'components/bootstrap';
+import DataNodeStatusCell from 'components/datanode/DataNodeList/DataNodeStatusCell';
+import { Link } from 'components/common/router';
+import Routes from 'routing/Routes';
+import DataNodeActions from 'components/datanode/DataNodeList/DataNodeActions';
+
+import type { ClusterNodes } from './useClusterNodes';
+import ClusterStatusLabel from './ClusterStatusLabel';
+import ClusterActions from './ClusterActions';
+
+type Props = {
+  clusterNodes: ClusterNodes,
+}
+
+const getRoleLabels = (roles: string) => roles.split(',').map((role) => (
+  <span key={role}>
+    <Label bsSize="xs">{role}</Label>&nbsp;
+  </span>
+));
+
+const ClusterConfigurationListView = ({ clusterNodes }: Props) => (
+  <Table>
+    <thead>
+      <tr>
+        <th>Node</th>
+        <th>Type</th>
+        <th>Role</th>
+        <th>State</th>
+        <th className="text-right">Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      {clusterNodes.graylogNodes.map((graylogNode) => (
```

Comment:
When loading the page, the list is empty for a bit while waiting for the API response. We should show a Loader to let users know that something’s happening. 
![cluster overview](https://github.com/user-attachments/assets/1867fd66-a475-456d-970f-4edccf9a0076)